eisbuk-admin/src/components/atoms/AttendanceCard/AttendanceCard.tsx

```diff
@@ -17,17 +19,15 @@ const AttendanceCard: React.FC<Props> = ({
   userBookings,
   absentees,
 }) => {
-  // convert timestamp to luxon for easier processing
+  const dispatch = useDispatch();
+
   const luxonStart = fb2Luxon(date);
 
-  // convert durations to number values
   const durationNumbers = durations.map((duration) => Number(duration));
   const longestDuration = Math.max(...durationNumbers);
 
-  // get end time of longest duration (we're still using durations here, so it's pretty straightforward)
   const luxonEnd = luxonStart.plus({ minutes: longestDuration });
 
-  // get time for rendering
```

Comment:
It's perfectly ok to leave comments. Just pull the first one to be above the `luxonStart`